src/the_program.cpp

```diff
@@ -22,6 +22,23 @@ void c_the_program::startup_console_first() {
 
 /// Show program version
 void c_the_program::startup_version() {
+
+
+
+
+
+
+	{
```

Comment:
hmmm

![why-am-i-a-camel](https://user-images.githubusercontent.com/18722764/27082135-d1672ffc-5043-11e7-9b2a-938f0c37113f.jpg)

Oh right, was testing how to get UBSAN to show proper backtrace.

:ok_hand: